mls-rs-core/src/crypto/cipher_suite.rs

```diff
@@ -81,3 +81,17 @@ impl CipherSuite {
         (1..=7).map(CipherSuite)
     }
 }
+
+/// Modes of HPKE operation.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[repr(u8)]
+pub enum HpkeModeId {
```

Comment:
Should it be `#[non_exhaustive]` or are we sure new variants will never be added?

Reply:
They are defined in the [RFC table](https://www.rfc-editor.org/rfc/rfc9180.html#table-1) and they don't seem to be extensible but I don't mind adding the flag. There may be more such places, I can double check in a follow up PR?

Reply:
Lets do a follow up